lang/java/reef-examples/src/main/java/org/apache/reef/examples/hello/HelloTask.java

```diff
@@ -21,6 +21,9 @@
 import org.apache.reef.task.Task;
```

Comment:
Undo all changes to this file. I had changed it for testing and these should not be there in the check in.

Reply:
done.